.gitignore

```diff
@@ -1,3 +0,0 @@
-/target
-**/*.rs.bk
-Cargo.lock
```

Comment:
Unintended change?

Reply:
I don't know how this sneak into here...